rawx-lib/src/attr_handler.c

```diff
@@ -168,6 +196,57 @@ _get (int fd, const char *k, gchar **pv)
 
 #define GET(K,R) _get(fd, ATTR_DOMAIN "." K, &(R))
 
+static GString*
+_search_user_oio(gchar* prefix, gchar * xattrlist, ssize_t size, gchar *name)
```

Comment:
What this function does is not obvious. You should rename it properly and document it with an example xattr list and what you look for.